book/website/project-design/pd-bias/pd-bias-identification.md

```diff
@@ -0,0 +1,89 @@
+
+The exposure can be anything which varies within the population, such as different treatments given in medical research, or smoking vs non-smoking, or different strategies for forest management. 
+The term 'intervention' may be used to refer to exposures which are actively applied rather than observed, such as different treatments in a medical trial. 
+
+The outcome can be anything which is measurable, either by qualitative (descriptive) or quantitative (numerical) means. 
+In answering the research question, the exposures and outcomes must be clearly defined, and applied/assessed consistently for all participants. 
+If the outcome is assessed qualitatively, it is also important that it is assessed without knowledge of the predictors to avoid an *observer bias*. 
+
+(pd-bias-identification-analysis)=
+## Analysis and Reporting
+Bias remains an issue after the data collection stage, with potential risks in the data analysis and reporting of results. 
+
+Weak validation can lead to bias due to overly optimistic results.
+For example, if results are generated using a single small test set then they may be better than expected (optimistic) simply due to random chance. 
+This is especially true if many different experiments are conducted, with a large enough number of experiments guaranteeing at least one positive result due to random chance. 
+This is sometimes referred to as 'p-hacking' in a statistical context where positive results have small p-values, and as such it is not always sufficient to use the standard p-value threshold of 0.05 (for example, [genome-wide association studies](https://academic.oup.com/g3journal/article/11/2/jkaa056/6080665) often use a threshold of 0.00000005, which can be written more clearly as 5x10<sup>-8</sup>). 
+
+Weak validations give imprecise results which are unlikely to generalise well to data outside of the training set. Thorough validation methods are explored in the 
+{ref}`validation and generalisability subchapter<pd-bias-validation>`. 
+
+It is also important that research is reported in a fair and unbiased manner. 
+*Reporting bias* typically involves placing undue emphasis on positive results while downplaying or ignoring negative results (also called *selective reporting* or *cherry picking*). 
+Failure to report negative results can be a significant form of scientific misconduct, and in medical research it is a breach of the [Declaration of Helsinki](https://www.wma.net/policies-post/wma-declaration-of-helsinki-ethical-principles-for-medical-research-involving-human-subjects/). 
+*Reporting bias* can also include bias in choosing which previous studies to cite in order to give unfair emphasis to a particular narrative. 
+
+To avoid biases in reporting research, it is helpful to define the primary results are before conducting research, then these can be reported regardless of whether the finding was positive or negative. 
```

Comment:
```suggestion
To avoid biases in reporting research, it is helpful to define what the primary results are before conducting research, then these can be reported regardless of whether the finding was positive or negative. 
```
I think what is missing here?